src/components/TrackTile/TrackerRow/Tracker.tsx

```diff
@@ -43,7 +45,7 @@ export function Tracker(tracker: TrackerProps) {
         <View style={styles.trackerProgressBarContainer}>
           <RadialProgress
             disabled={disabled}
-            color={tracker.color}
+            color={theme.colors.primary}
```

Comment:
Just want to double check and make sure that we want all tracker progress indicators to use the primary theme color.